gamestonk_terminal/feature_flags.py

```diff
@@ -1,12 +1,23 @@
 import os
 
-USE_COLOR = os.getenv("GTFF_USE_COLOR") or True
-USE_FLAIR = os.getenv("GTFF_USE_FLAIR") or "stars"
-USE_ION = os.getenv("GTFF_USE_ION") or True
-USE_PROMPT_TOOLKIT = os.getenv("GTFF_USE_PROMPT_TOOLKIT") or False
+# env vars are strings not booleans
+# check for "false" or "0"
+# any other value is true (including empty string)
+
+USE_COLOR = os.getenv("GTFF_USE_COLOR", "true").lower() not in ["false", "0"]
+USE_FLAIR = os.getenv("GTFF_USE_FLAIR", "stars")
+USE_ION = os.getenv("GTFF_USE_ION", "true").lower() not in ["false", "0"]
+USE_PROMPT_TOOLKIT = os.getenv("GTFF_USE_PROMPT_TOOLKIT", "false").lower() not in [
+    "false",
+    "0",
+]
+
 
 # Enable Prediction features
-ENABLE_PREDICT = os.getenv("GTFF_ENABLE_PREDICT") or False
+ENABLE_PREDICT = os.getenv("GTFF_ENABLE_PREDICT", "false").lower() not in ["false", "0"]
```

Comment:
Why did you add this?

Reply:
Could it be that in the environment variable you were using false as a string, rather than the bool "False"? Asking because I'm curious :)

Reply:
I also don't think this is necessary and looks very confusing.

```ENABLE_PREDICT = os.getenv("GTFF_ENABLE_PREDICT") or False``` will be a non-empty string if an environment variable is set or False if an environment variable is not set. Python behavior for non-empty strings is the same behavior as for the True value in the conditional statement. That is all that we need.

Reply:
This is correct for options that default to False. However, I don't think the same pattern works for options that default to True, which would look like `ENABLE_FOO = os.getenv("GTFF_ENABLE_FOO") or True`. There's nothing you can set an environment variable to in that case that will cause ENABLE_FOO to evaluate to False. However, at the time I made those changes I wasn't aware that, per your comment, the tests aren't importing the feature flags at all. I totally agree that it's ugly and dumb, but one of the PRs that's blocked behind this one sets prompt toolkit as enabled by default, and if we can't then turn it off with an environment variable we're right back to the original problem.

Reply:
Regarding the type of the environment variable: from the OS level, you can't set a type. os.getenv returns a string or whatever value is specified in the default kwarg (set to None if unspecified) if it can't find the key in the environment. Python evaluates "", 0, and None as falsey and pretty much everything else is truthy, and that's the conversion going on when we're doing comparisons against the result of os.getenv.

That said, I would absolutely welcome a cleaner and less ugly way of doing this, I'm just not sure what that looks like.

Reply:
Let me have a got at this tomorrow, and I'll let you know :)

Reply:
> There's nothing you can set an environment variable to in that case that will cause ENABLE_FOO to evaluate to False.

That's a great point and something I missed.

Reply:
OK, there is a better way to do this, Python ships a [strtobool function in distutils.utils](https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool) that does exactly what I'm trying to do with the awkward if not in [] construct. I'll use that instead.

Reply:
Check this out - https://github.com/DidierRLopes/GamestonkTerminal/pull/230

Reply:
> Check this out - #230

I'll leave it up to you and @DidierRLopes as maintainers to decide whether you prefer the version I already pushed or this one, but this one might have some unexpected behavior, particularly regarding integration with other (Unix) tools that output '1' and '0' or anything that converts strings to lowercase. `"true`, `'1'`, and `1` all evaluate as False with that patch but are handled as expected with `distutils.util.strtobool()`.

Reply:
I like your approach. Want to go with it?

Reply:
I'll go ahead with the merge.

Thank you for great work!

Reply:
Great work on this @clothespin . Thank you!